creator.html

```diff
@@ -174,7 +168,7 @@ <h6 class="feature-title">Material Design</h6>
                         </div>
                         <div class="col-10">
                             <h6 class="feature-title">Fully compatible</h6>
-                            <p class="grey-text">All Android phones and tablets are supported from Android 4.1 JellyBean! There are no restrictions</p>
+                            <p class="grey-text">We support all android phones and tablets ranging from Android 4.1 JellyBean! There are no restrictions.</p>
```

Comment:
Android should be always be capitalised.

```suggestion
                            <p class="grey-text">We support all Android phones and tablets ranging from Android 4.1 JellyBean! There are no restrictions.</p>
```